react/Button/Button.less

```diff
@@ -75,10 +97,45 @@
   .buttonColor(@sk-black, @sk-mid-gray-light);
 }
 
+.root_isGhost,
 .root_isTransparent {
-  color: @sk-link;
   background-color: transparent;
   box-shadow: none;
+}
+
+.root_isGhost {
+  border: 2px solid;
+  font-weight: @sk-bold;
+  line-height: 0;
+}
+
+.root_isBlue.root_isGhost {
+  .ghostColor(@sk-link)
+}
+
+.root_isWhite.root_isGhost {
```

Comment:
The prop type validation here (https://github.com/seek-oss/seek-style-guide/pull/377/files#diff-bdfcd58f19e20b24ebd558c8a351c45fL13) needs to allow for white now

Reply:
Great pickups @georgespyropoulos 👍 Thanks, fixed